charts/opentelemetry-operator/templates/clusterrole.yaml

```diff
@@ -36,6 +36,9 @@ rules:
     resources:
       - namespaces
     verbs:
+      {{- if .Values.manager.createRbacPermissions }}
+      - create
+      {{- end }}
```

Comment:
@thefirstofthe300 this feels tangential to the `--create-rbac-permissions`.  I don't see where setting this flag in the operator provides this specific access. Do you have a link to the operator code/docs that is requiring this access?